vault/request_handling.go

```diff
@@ -261,7 +261,7 @@ func (c *Core) handleRequest(req *logical.Request) (retResp *logical.Response, r
 
 // handleLoginRequest is used to handle a login request, which is an
 // unauthenticated request to the backend.
-func (c *Core) handleLoginRequest(req *logical.Request) (*logical.Response, *logical.Auth, error) {
+func (c *Core) handleLoginRequest(req *logical.Request) (retResp *logical.Response, retAuth *logical.Auth, retErr error) {
```

Comment:
Isn't it confusing if we have both dedicated return variables and explicit return statements? I am not sure about the right way (idiomatic way) to go here. But, felt like I should bring attention to this once.